### DISCUSSION TITLE
How to Access Open Store Back Office page in DNN 9.11.1?

### DISCUSSION THREAD

**Discussion by mayaco** [openstore-ecommerce/OpenStore#291] [Q&A]
Hello,

I've installed the latest version of NB Store 4.1.11 on a clean DNN 9.11.1 and the documentation is a bit outdated. The Admin > Open Office page does not appear in the Persona Bar for DNN 9.11.1. Can anyone provide the URL to access the Open Store Back Office page?

**Comment by jasclar** [comment]
try  https://www.whatever/Admin/OpenStore-Admin

**Comment by mayaco** [comment]
Thank you for the suggestion, although that didn't work. I was able to figure it out. For the Admin Open Office page, I had to assign the OpenStoreBO theme to it. Once I did that, browsing to that page brought up the Admin Open Office page.
